Assets/MRTK/SDK/Features/UX/Scripts/Pointers/TeleportPointer.cs

```diff
@@ -411,13 +410,9 @@ public override void OnPostSceneQuery()
                                     : TeleportSurfaceResult.Invalid;
                             }
                         }
-                        else if (((1 << Result.CurrentPointerTarget.layer) & InvalidTeleportationLayers) != 0)
```

Comment:
We removed this check because in the new paradigm we don't need the last case. 

We don't use it anywhere else, but I didn't want to break the interface since anyone who subclassed the teleport pointer might rely on it.